com.trekglobal.idempiere.rest.api/keycloak/Testing Steps.txt

```diff
@@ -0,0 +1,76 @@
+1. Install keycloak at port 9080 with password admin/admin
+- sudo docker run -p 9080:8080 -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin quay.io/keycloak/keycloak:21.1.1 start-dev
```

Comment:
suggesting to change this to:

```
docker run -p 9080:8080 -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin quay.io/keycloak/keycloak:latest start-dev
```

Reply:
Exact version help to ensure the testing instruction is valid (some details could change between version). Anyway, the instruction is just for testing and not something for production use.